runtime/development/src/lib.rs

```diff
@@ -784,10 +785,19 @@ impl pallet_loan::Config for Runtime {
 	type Time = Timestamp;
 	type LoanPalletId = LoanPalletId;
 	type Pool = InvestorPool;
+	type Permission = Permissions;
 	type WeightInfo = pallet_loan::weights::SubstrateWeight<Self>;
 	type MaxLoansPerPool = MaxLoansPerPool;
 }
 
+impl pallet_permissions::Config for Runtime {
+	type Event = Event;
+	type Location = PoolId;
+	type Role = pallet_loan::PoolRole;
+	type Storage = PermissionRoles;
+	type AdminOrigin = EnsureProportionAtLeast<_1, _2, AccountId, CouncilCollective>;
```

Comment:
Might make more sense to require Root here, and the council can only fast-track? I'm really unsure of the reasons why we'd want something to be deomcracy-able vs council-able

Reply:
Ah. Yes definitely. This was just to please the compiler as fast as possible. I have an open PR already, that I will merge that takes care of ensuring we always have root in place. I would prefer to go the kusama/polkadot route, that democracy should be able to do anything as admin as long as it is not distinct to be a council call, like the specific origin (forgot the name) that is defined in democracy.

Reply:
it should ideally be Council with fallback to Root.
For council, lets use same proportion we use for fast track in their respective runtimes